docs/preview/features/secret-store/index.md

```diff
@@ -12,6 +12,21 @@ Once register, you can fetch all secrets by using `ISecretProvider` which will g
 
 > :bulb: See [this section](#using-secret-store-within-azure-functions) if you want to use the secret store functionality whitin Azure Functions.
 
+## Why would I use it?
+Why would you use our Arcus secret store instead of just using the Azure SDK directly to access Azure Key Vault secrets?
+
+The Arcus secret store has some advantages over using the Azure SDK or configuration directly:
+
+✔ **Caching**: We provide caching so the secret providers will not be called upon every secret retrieval. This helps you avoiding hitting service limitations and we provide [asynchronous cache invalidation](https://background-jobs.arcus-azure.net/features/security/auto-invalidate-secrets).
+
+✔ **Plug & play**: We support using multiple and combinations of secret providers so with a single secret retrieval can query multiple secret providers (also multiple Azure Key Vaults).
+
+✔ **Design for security**: 
+- While using configuration for storing secrets can be good for development it is not a safe approach. With the secret store, we provide a single place to retrieve secrets instead of scattering the integration across the application. 
```

Comment:
Nit - Either we use bullet point for all or not, but I like this change

Reply:
Yes, I change this to bullet points.

Reply:
My suggestion was more to align the rest to be consistent

Reply:
Ah, like that. Yeah, that would be better. Then we can still use ✔, which I kinda like in this context 😄 .